jscomp/core/js_dump.ml

```diff
@@ -757,6 +757,20 @@ and expression_desc cxt ~(level : int) f x : cxt =
             (if !Js_config.debug then [ (name_symbol, E.str p.name) ] else [])
             (fun i -> Js_op.Lit i)
         in
+        let is_optional (names: string list) (pname: Js_op.property_name)  =
```

Comment:
No point in passing `names`, one can use directly `p.optional_labels`.

Reply:
Thanks https://github.com/rescript-lang/rescript-compiler/pull/5827/commits/c3f45453d4b773657680fd24cd6e46d391fa926f